docs/source/recipes/images.md

```diff
@@ -0,0 +1,51 @@
+# Images
+
+Volto comes with an Image component that allows to load images in an optimal way.
+
+Features of the Image component are:
+
+- optimized lazy loading
+- automatic `srcset` generation, if given a content item or catalog brain
+- optional responsive width
+
+The component can accept all the regular html `<img>` attributes, as well as a few extra for the features described above.
+
+Setting the `loading="lazy"` attribute to the Image component also adds the `decoding="async"` attribute which is generally considered an improvement when loading images lazily.
+
+In order to render an image from a content object, you would use the component like this:
+
+```javascript
```

Comment:
The JavaScript lexer is OK, but let's try JSX for React components. It usually does a better job, except when it cannot parse the code.

````suggestion
```jsx
````